hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -152,7 +152,7 @@ def main():
           "cover_rate"  : "90 %"
         },
         If coverage is enabled, this script will also parse the coverage result:
-        "fpv_coverage": {
+        "formal_coverage": {
```

Comment:
Does FPV and conn generate the same coverage metrics?
I feel connectivity should generate toggle coverage which is more useful. 
That would of course, mean this script will then need to understand what sub_flow and tool are being used.

Reply:
Thanks Sri for the pointing that out! I think right now when we enable coverage, we got the same result as FPV [connectivity coverage](https://www.cadence.com/en_US/home/tools/system-design-and-verification/formal-and-static-verification/jasper-gold-verification-platform/jaspergold-design-coverage-verification-app.html), and the toggle coverage is right now included in "stimuli coverage". I am working on separate it out and yes, we will need to input {sub_flow} name for this file to parse correct coverage.
I have been having some license acquiring issue, so have not pushed a PR for that yet..